### PR TITLE
add mmap implement the interface IOManager

### PR DESCRIPTION
**Pull request by ZhaoShuaiUp**
Test Result:
Read Speed:    file_io(0.65 s) VS mmap_io(0.34s)
Write Speed:   file_io(1.47 s) VS mmap_io(0.36s)

### CODE REVIEW

File: fio/mmap_io.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+// MMapIO 标准系统文件IO
+type MMapIO struct {
+	fd     *os.File // 系统文件描述符
+	data   []byte   // 与文件对应的映射区
+	dirty  bool     // 是否更改过
+	offset int64    // 写入位置
+}
+
+// NewMMapIOManager 初始化标准文件 IO
+func NewMMapIOManager(fileName string) (*MMapIO, error) {
```
Comment (sjcsjc123) [inline fio/mmap_io.go:22]: image
you can replace it with this and test the other `db` tests can run or not

Comment (sjcsjc123) [reply fio/mmap_io.go:22]: and show the benchmark test result
image

Comment (ZhaoShuaiUp) [reply fio/mmap_io.go:22]: the benchmark test is pass, 1879 ns/op(mmap IO) VS 3181 ns/op(file IO)
C29`~AVA{4T3PGJNI7GN4SQ

NC $7{G0{1WX~LK4S12( C1

Comment (ZhaoShuaiUp) [reply fio/mmap_io.go:22]: Benchmark TestPutAndGet result（50w operations）

7dd1bbe4ef8581af51ce0d4fcb834b9

**Pull request merged**